pkg/providers/ovirt/validation/validators/vm-validator.go

```diff
@@ -5,10 +5,15 @@ import (
 	"strconv"
 
 	ovirtsdk "github.com/ovirt/go-ovirt"
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
 )
 
 // BiosTypeMapping defines mapping of BIOS types between oVirt and kubevirt domains
-var BiosTypeMapping = map[string]string{"q35_ovmf": "efi", "q35_sea_bios": "bios", "q35_secure_boot": "bios"}
+var BiosTypeMapping = map[string]*kubevirtv1.Bootloader{
```

Comment:
What do you think about keeping it in the mapper and referencing it in the validation?

Reply:
Well, since it's called `BiosTypeMapping` looks like a better package is `mapping` :) so I've moved it.